src/SprykerMiddleware/Shared/Process/Log/MiddlewareLoggerTrait.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerMiddleware\Shared\Process\Log;
+
+use Psr\Log\LoggerInterface;
+use Spryker\Shared\Log\Config\LoggerConfigInterface;
+use Spryker\Shared\Log\LoggerTrait;
+use SprykerMiddleware\Shared\Process\Log\Exception\ProcessLoggerNotConfiguredException;
+
+trait MiddlewareLoggerTrait
```

Comment:
Could it be extracted to a separate module, to make the logging consistent along all middleware modules?